certif/app/controllers/authenticated/sessions/details/certification-candidates.js

```diff
@@ -105,7 +105,7 @@ export default class CertificationCandidatesController extends Controller {
       }
       await certificationCandidate
         .save({ adapterOptions: { registerToSession: true, sessionId } });
-      this.model.certificationCandidates.insertAt(0, certificationCandidate);
+      this.currentSession.certificationCandidates.insertAt(0, certificationCandidate);
```

Comment:
[BSR] [unshiftObject](https://api.emberjs.com/ember/2.14/classes/Ember.MutableArray/methods/unshiftObject?anchor=unshiftObject)
```suggestion
      this.currentSession.certificationCandidates.unshiftObject(certificationCandidate);
```